macro/o2sim.C

```diff
@@ -106,6 +107,8 @@ void o2sim()
   rtdb->saveOutput();
   rtdb->print();
 
+  o2::PDG::addParticlesToPdgDataBase(0);
```

Comment:
@shahor02 : This looks very nice. Just few small questions:

a) is PDG maybe a too short a name and why should it be under Generators?
b) Maybe we can put the addition into some more fundamental class which is always called? Like our stack?
Like this we make sure that someone (not using o2sim for tests) misses it.

Reply:
@sawenzel To me the PDG is as unambiguous as e.g. HEP, but if you want to propose other name, fine. As for where to put it, I was not sure and put in Generators for the sake of logical connection to them. Tell me where to put them, I will move it.
For the addition: we can put it in the stack constructor, but are we sure we will have just one copy of the stack (e.g. in case of overlaid events?). The class has an internal protection to not add the same particle twice, but these checks slow down the initialization.

Reply:
Ok. Let's merge as it is to unblock further development.